assets/javascripts/discourse/components/ai-llm-editor.gjs

```diff
@@ -1,306 +1,89 @@
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
-import { Input } from "@ember/component";
-import { on } from "@ember/modifier";
 import { action } from "@ember/object";
-import { LinkTo } from "@ember/routing";
-import { later } from "@ember/runloop";
-import { inject as service } from "@ember/service";
-import { or } from "truth-helpers";
 import BackButton from "discourse/components/back-button";
 import DButton from "discourse/components/d-button";
-import DToggleSwitch from "discourse/components/d-toggle-switch";
-import Avatar from "discourse/helpers/bound-avatar-template";
-import { popupAjaxError } from "discourse/lib/ajax-error";
-import icon from "discourse-common/helpers/d-icon";
-import i18n from "discourse-common/helpers/i18n";
 import I18n from "discourse-i18n";
-import AdminUser from "admin/models/admin-user";
 import ComboBox from "select-kit/components/combo-box";
-import DTooltip from "float-kit/components/d-tooltip";
+import AiLlmEditorForm from "./ai-llm-editor-form";
 
 export default class AiLlmEditor extends Component {
-  @service toasts;
-  @service router;
-  @service dialog;
+  @tracked ranWizard = false;
+  preConfiguredLlm = "none";
 
-  @tracked isSaving = false;
-
-  @tracked testRunning = false;
-  @tracked testResult = null;
-  @tracked testError = null;
-  @tracked apiKeySecret = true;
-
-  get selectedProviders() {
-    const t = (provName) => {
-      return I18n.t(`discourse_ai.llms.providers.${provName}`);
-    };
-
-    return this.args.llms.resultSetMeta.providers.map((prov) => {
-      return { id: prov, name: t(prov) };
-    });
-  }
-
-  get adminUser() {
-    return AdminUser.create(this.args.model?.user);
+  get showWizard() {
```

Comment:
Would love to leave out `wizard` terminology here, don't want to set any precedents yet :laughing: 

Suggest `showPresets` here and `chosePreset` instead of `ranWizard`

Reply:
will change